prow/scripts/cluster-integration/kyma-gke-long-lasting.sh

```diff
@@ -350,7 +350,7 @@ log::info "install image-guard"
 helm install image-guard "$TEST_INFRA_SOURCES_DIR/development/image-guard/image-guard"
 
 export INSTALL_DIR=${TMP_DIR}
```

Comment:
you can remove this line. This was used by the Kyma-cli script for installation the binary to temporary directory.